frontend/public/components/monitoring.tsx

```diff
@@ -1415,7 +1415,7 @@ const AlertingPage: React.SFC<AlertingPageProps> = ({ match }) => {
         <h1 className="co-m-pane__heading">
           <div className="co-m-pane__name co-resource-item">
             <span className="co-resource-item__resource-name" data-test-id="resource-title">
-              Alerting
+              Alertmanager
```

Comment:
This will also change the title for the Monitoring Alerts and Silences list pages, not just the Alertmanager config pages.